code/modules/projectiles/guns/projectile/revolver.dm

```diff
@@ -107,13 +107,13 @@
 	name = "\"The Judge\""
 	desc = "A revolving hand-shotgun by Cybersun Industries that packs the power of a 12 guage in the palm of your hand (if you don't break your wrist). Uses 12 shotgun rounds."

```

Comment:
I'm just going to give my usual "shut up I'm sleepy" rebuttal. I was probably sleepy at the time.